### PR TITLE
Add missing architecture-dependent binaries to package

### PR DESCRIPTION
**Pull request by borchero**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by borchero** [comment]
@conda-forge-admin, please rerender

**Comment by borchero** [comment]
@conda-forge-admin, please rerender

**Comment by maresb** [comment]
What's the motivation for this? Installing from wheels is quite non-standard for conda-forge.

**Comment by borchero** [comment]
@maresb I don't know if you've used this conda package recently but it does not actually work. Quite a few binaries are missing that are only available in the wheels that can be found on PyPi.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,14 +5,12 @@ package:
...
-  url: https://github.com/microsoft/mssql-scripter/archive/v{{ version }}.tar.gz
-  sha256: 863b820f4ddd0715d01ae7c38047b9396d35fb705222bbaeae5f59e708c0cfa0
-
 build:
   noarch: python
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 1
+  script: "{{ PYTHON }} -m pip install https://files.pythonhosted.org/packages/d0/49/e3a6699c0773063fd8a443fd82b9faa0b8d78f01d75ac5c2327be15f4afe/mssql_scripter-{{ version }}-py2.py3-none-win_amd64.whl -vv"  # [win64]
+  script: "{{ PYTHON }} -m pip install https://files.pythonhosted.org/packages/77/d1/efa8774ccb2a542567f777bf2a6749d4fcd868477edb3deba9a968d5f62c/mssql_scripter-1.0.0a23-py2.py3-none-manylinux1_x86_64.whl -vv"  # [linux]
```
[the remark this answers is not in the document]
Comment (borchero) [reply recipe/meta.yaml:12]: Oh yes, sure ... this PR is far from ready anyway 😬

**Comment by maresb** [comment]
Hmm, I haven't used it recently. I'm sorry to hear that it's broken.

The conda-forge philosophy is that everything should be compiled on the conda-forge infrastructure. Are the missing libraries open-source?

**Comment by borchero** [comment]
> The conda-forge philosophy is that everything should be compiled on the conda-forge infrastructure. Are the missing libraries open-source?

I'm checking ... I realized that downloading the wheel is also very undesirable because it enforces a useless dependency on `enum34` because the wheel was created with an ancient version of Python apparently 😬

**Comment by borchero** [comment]
@conda-forge-admin, please rerender

**Comment by borchero** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,16 +10,15 @@ source:
   sha256: 863b820f4ddd0715d01ae7c38047b9396d35fb705222bbaeae5f59e708c0cfa0
 
 build:
-  noarch: python
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 1
+  merge_build_host: true  # workaround conda-build bug on windows
```
Comment (xhochy) [inline recipe/meta.yaml:14]: What error message are you getting?

Comment (borchero) [reply recipe/meta.yaml:14]: See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=686673&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=1183ba29-a0b5-5324-8463-2a49ace9e213&l=324. Last time I observed this, this was the key which fixed it (and it did again).

**Pull request merged**